src/App.vue

```diff
@@ -293,6 +324,16 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
       console.log('Error while retrieving NR during File and Pay') // eslint-disable-line no-console
       this.nameRequestInvalidErrorDialog = true
     })
+
+    // Set identifier to store
+    const id = this.$route.query?.id as string
+    if (id?.startsWith('CP') || id?.startsWith('BC')) {
+      this.setBusinessId(id)
+      this.setFilingType(FilingTypes.DISSOLUTION)
```

Comment:
Here you call it Filing Type, but above the enum is Filing Name. Is it all the same thing? Does this need both store properties?

Reply:
Two different properties and values. 
 Filing Name for instance `Incorporation Application` 
 Filing type as lear defines it `incorporationApplication`
 and so on. 
 So the filing name is most often used for display purposes and the type is the data.